### DISCUSSION TITLE
Participate in Hacktoberfest?

### DISCUSSION THREAD

**Discussion by markdav-is** [oqtane/oqtane.framework#782] [General]
Hacktoberfest is happening all month:  https://hacktoberfest.digitalocean.com/

If @sbwalker is looking for more contributors, we could add the hacktoberfest topic to the repo as well as some approachable issues, like documentation!  

more details here: https://www.youtube.com/watch?v=FxBv4FMcDgo&ab_channel=DigitalOcean

**Comment by kencutt** [comment]
Hi @markdav-is ....I like that idea as well as investigating adding more automated tests to the test project for unit testing changes to the project.  Thoughts?

**Reply by markdav-is** [reply]
sounds reasonable.  I feel like we need permission from @sbwalker to participate?  the PRs need to be accepted to count for the contest.

**Reply by sbwalker** [reply]
My apologies for the late response here... as part of my role as Chair of Project Committee for .NET Foundation we discussed Hacktoberfest in great detail for all projects in the Foundation. Unfortunately Hacktoberfest resulted in a lot of repository spam this year for maintainers of open source projects. People would submit random/invalid PRs to repos and the responsibility is then on the maintainers/community to police the spam. Many projects opted out as a result of this because it did not achieve the goal of attracting the right type of contributions.

**Reply by markdav-is** [reply]
right!  it takes a bit of work to groom the simple issues that folks could pick up as part of the PR.  it's also an administrative load that that would take time away from the core efforts.  A better idea for another year when we have more cooks.
